src/Models/Link.php

```diff
@@ -62,6 +62,11 @@ class Link extends DataObject
      */
     private static int $menu_priority = 100;
 
+    /**
+     * Set the default icon for the link type
+     */
```

Comment:
```suggestion
    /**
     * The css class for the icon to display for this link type
     */
```
It's not directly an icon - it's a css class.
And it's not the "default" css class, it's _the_ css class for the icon.

Reply:
Updated